.circleci/config.yml

```diff
@@ -335,10 +334,63 @@ jobs:
             https://api.github.com/repos/NCAR/VAPOR/issues/${CIRCLE_PULL_REQUEST##*/} \
             -d "$json"
 
-  build_ubuntu18:
+  python_api_centos:
+    docker:
+      - image: sgpearse/vapor3-centos7:latest
+
+    resource_class: large
+
+    steps:
+      - run:
+          name: update 3rd party libraries
+          command: |
+            rm -rf /usr/local/VAPOR-Deps/2019-Aug
+            pip3 install gdown --upgrade
+            gdown https://drive.google.com/uc?id=1S9DwySMnQrBuUUZGKolD__WQrjTmLgyn
+            filename="/root/project/2019-Aug-CentOS.tar.xz"
+            bsdtar -xf ${filename} -C /usr/local/VAPOR-Deps
+
+      - run:
+          name: install python
+          command: |
+            yum -y update || true
+            yum install -y python3
+            yum install -y python3-pip
+            pip3 install --upgrade pip
+            pip3 install scikit-build command
+
+      - run:
+          name: install miniconda
```

Comment:
You could just use the miniconda docker image.

Reply:
I didn't know they had one based on debian.  Sounds good.